app/assets/javascripts/backbone/views/page_view.coffee

```diff
@@ -0,0 +1,12 @@
+class MS.Views.PageView extends Backbone.View
+
+  el: 'body'
+
+  events: (params) ->
+    'change select[name="division_id"]': 'selectDivision'
+
+  selectDivision: (e) ->
+    $item = $(e.currentTarget)
+    $("input[name=redisplay_url]").val(window.location)
```

Comment:
Currently the selected division controls the filtering applied to the index view, but it doesn't change any access control logic, so it's not going to trigger any 404 errors.  Do we want some different behavior around this?